model/registry/file.yaml

```diff
@@ -4,12 +4,62 @@ groups:
     type: attribute_group
     brief: "Describes file attributes."
     attributes:
+      - id: accessed
+        type: string
+        brief: >
+          Time when the file was last accessed, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: attributes
+        type: string[]
+        brief: >
+          Array of file attributes.
+        note: >
+          Attributes names will vary by platform. Here’s a non-exhaustive list of values that are expected in this
+          field: archive, compressed, directory, encrypted, execute, hidden, read, readonly, system, write.
+        stability: experimental
+        examples: ['readonly', 'hidden']
+      - id: created
+        type: string
+        brief: >
+          Time when the file was created, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: ctime
+        type: string
+        brief: >
+          Time when the file attributes or metadata was last changed, in ISO 8601 format.
+        note: >
+          This attribute is not supported on all file systems. Note that changes to the
+          file content will update `mtime`. This implies `ctime` will be adjusted at the same time, since
+          `mtime` is an attribute of the file.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
+      - id: device
```

Comment:
We have `device.id` already in the registry https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/device.md. I think it makes sense to re-use it, no?

Reply:
hey @joaopgrassi I understand that they are connected but I think this particular device is connected to the file.
We have similar situation as in [log PR](https://github.com/open-telemetry/semantic-conventions/pull/908#pullrequestreview-2003998285) - when we will be able to embed namespace into another namespace, in this case `device` under `file`, we could use this field from device namespace. But in current situation I think it's important that we have it as is.

Reply:
Not sure I got it. Even if we could embed it would still be `device.id` no? Is the problem because ECS defines it as `device` only and it's not desired to change it to `device.id`? I'm just a bit worried that now there will be another "device" out there 🤔

Reply:
if we expected multiple devices to be present on the same telemetry item it would make sense to define `file.device.id`.  Is it a common case here?

Otherwise having multiple attributes would not allow users to correlate/query data base on attribute value.

Reply:
if it will be embedded it will be `file.device`
If we have multiple files notated in log entry, each of the files could have it's own `device`.

But I think I better skip this field for now until re-use will be there and later we could discuss if we need to add it or not

Reply:
I don't think the Linux Device has the same meaning as the existing semconv device. Linux devices OS-level representation of hardware peripherals that are accessible to the OS. In the context of files, device represents e.g the hard drive that the file resides in. Reading the existing semconv device page, it semms to be referring to the source of telemetry, like a mobile phone, or computer or something. Both should probably called "device" but perhaps it's ok to duplicate it instead of reference, since the semantics are different, and therfore the attributes will be different, validation rules, etc.

Reply:
Hey @itaysk. Registry is meant to be the source of truth for all our observed data and it should be agnostic. 
Previously there were use cases for mostly observability but now we are expanding it for other domains as well. It means if we see that registry is not generic enough it should be updated to support it. In this case add better description or examples. 
But I understand your point. I have omitted this field in this PR to discuss it separately and there we can come to resolution - use here directly as very specific to the file or use the reference from global device namespace. 
Does it resolve your doubts?